templates/raspberry-pi/index.html

```diff
@@ -0,0 +1,183 @@
+{% extends "raspberry-pi/_base_raspberry-pi.html" %}
+
+{% block title %}Ubuntu for Raspberry Pi{% endblock %}
+
+{% block meta_description %}The Raspberry Pi is an ARM computer for everybody. Perfect for teaching, coding, surfing the web or simply as a desktop for anyone, anywhere. The Raspberry Pi with Ubuntu is a gateway to the world of open source invention.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1XS8KyKQnQdgY9NJfxHMs_9ql4sPOTRj0yweunMPoOdM/edit#heading=h.alvyav8qzr5r{% endblock meta_copydoc %}
```

Comment:
This is not the right doc

Reply:
Should be: https://docs.google.com/document/d/1XS8KyKQnQdgY9NJfxHMs_9ql4sPOTRj0yweunMPoOdM/edit